packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -650,6 +686,36 @@ export default class Health extends Component {
             </Card>
           </Col>
         </Row>
+        {nonSharedDefaultSr.length > 0 && (
+          <Row>
+            <Col>
+              <Card>
+                <CardHeader>
+                  <Icon icon='disk' /> {_('nonSharedDefaultSr')}
+                </CardHeader>
+                <CardBlock>
+                  <NoObjects
```

Comment:
I think we should add a small tip there that says something like "It is usually recommended for a pool's default SR to be shared to avoid unexpected behaviors" so that it's clearer what the purpose of this section is.